app/client/views/application/application.html

```diff
@@ -76,21 +76,6 @@
           </select>
         </div>
 
-        <div class="field">
```

Comment:
Why is all of this in the pull req?

Reply:
i dont know, its not actually broken i have no clue what happened with the commit. i even cleared it and did it again and it did that? does it work for you?